pkg/collector/lv_collector.go

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright 2021 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package collector
+
+import (
+	"github.com/openebs/lvm-localpv/pkg/lvm"
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/klog"
```

Comment:
Consider following the import order, by providing spacing between packages in the same repo and vendored packages.

Reply:
Didn't get you @akhilerm .

Reply:
```suggestion
	"github.com/prometheus/client_golang/prometheus"
	"k8s.io/klog"
	
	"github.com/openebs/lvm-localpv/pkg/lvm"
```
by running the goimports tool